spec/cmds/status_spec.ts

```diff
@@ -33,7 +33,7 @@ describe('status', () => {
         .catch(err => {
           done.fail();
         });
-  });
+  }, 30000);
```

Comment:
Running update twice sometimes was taking more than Jasmine's default timeout (5s).
Updating this timeout to 30s made this test suite a lot more stable.